content/en/docs/application-store/external-apps/deploy-tidb.md

```diff
@@ -0,0 +1,217 @@
+
+### Step 4: Deploy a TiDB cluster
+
+The process of deploying a TiDB cluster is similar to deploying the TiDB Operator.
+
+1. Go to **Applications** under **Application Workloads** and click **Deploy New Application** again and then select **From App Templates**.
+
+   ![deploy-app-again](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/deploy-app-again.PNG)
+
+   ![from-app-templates-2](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/from-app-templates-2.PNG)
+
+2. From the PingCAP repository, click **tidb-cluster**.
+
+   ![click-tidb-cluster](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/click-tidb-cluster.PNG)
+
+3. On the **Chart Files** tab, you can view the configuration and download the `values.yaml` file. Click **Deploy** to continue.
+
+   ![download-yaml-file](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/download-yaml-file.PNG)
+
+4. On the **Basic Info** page, confirm the app name, app version, and deployment location. Click **Next** to continue.
+
+   ![tidb-cluster-info](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/tidb-cluster-info.PNG)
+
+5. Some TiDB components require [persistent volumes](../../../cluster-administration/persistent-volume-and-storage-class/). You can run the following command to view your storage classes.
+
+   ```bash
+   kubectl get sc
+   ```
+
+   In this regard, the following storage classes will be deployed automatically by default if you install KubeSphere through QingCloud Kubernetes Engine (QKE).
+
+   ```
+   / # kubectl get sc
+   NAME                       PROVISIONER     RECLAIMPOLICY   VOLUMEBINDINGMODE   ALLOWVOLUMEEXPANSION   AGE
+   csi-high-capacity-legacy   csi-qingcloud   Delete          Immediate           true                   71m
+   csi-high-perf              csi-qingcloud   Delete          Immediate           true                   71m
+   csi-ssd-enterprise         csi-qingcloud   Delete          Immediate           true                   71m
+   csi-standard (default)     csi-qingcloud   Delete          Immediate           true                   71m
+   csi-super-high-perf        csi-qingcloud   Delete          Immediate           true                   71m
+   ```
+
+   {{< notice note >}}
+
+   The QingCloud CSI plugin implements an interface between [CSI](https://github.com/container-storage-interface/)-enabled Container Orchestrator (CO) and the storage of QingCloud. For more information about QingCloud CSI, see the [GitHub repository](https://github.com/yunify/qingcloud-csi).
+
+   {{</ notice >}}
+
+6. On the **App Config** page, change the default value of the field `storageClassName` from `local-storage` to `csi-qingcloud`. In the downloaded file, you can replace all the values directly and copy and paste the modified local file into the `values.yaml` file on the console.
```

Comment:
change the default value of the field `storageClassName` from `local-storage` to `csi-qingcloud`.
->
change the default value of the field `storageClassName` from `local-storage` to the name of your storage class. For example, you can change it to `csi-qingcloud` based on the above output.